forecasts.js

```diff
@@ -366,75 +348,84 @@ forecasts.getRegionDetailsForRegionId = function(regionId) {
                     break;
                 case 'ipac':
                     var ipacPaths = {
-                        1: 'http://www.idahopanhandleavalanche.org/advisories/selkirks-cabinets-map-xml',
-                        2: 'http://www.idahopanhandleavalanche.org/advisories/selkirks-cabinets-map-xml',
-                        3: 'http://www.idahopanhandleavalanche.org/advisories/st-regis-silver-map-xml'
+                        1: 'https://www.idahopanhandleavalanche.org/advisories/selkirks-cabinets-map-xml',
+                        2: 'https://www.idahopanhandleavalanche.org/advisories/selkirks-cabinets-map-xml',
+                        3: 'https://www.idahopanhandleavalanche.org/advisories/st-regis-silver-map-xml',
+                        4: 'https://www.idahopanhandleavalanche.org/advisories/kootenai-map-xml',
+                        5: 'https://www.idahopanhandleavalanche.org/advisories/kootenai-map-xml'
```

Comment:
did kootenai move from FAC to IPAC?

Reply:
Yes. They moved over in the fall of 2017.

Reply:
perfect, thanks.